docs/userguide/src/portingguide/debugging/assertions.md

```diff
@@ -0,0 +1,15 @@
+# Enabling debug assertions
+
+MMTk is implemented with an extensive amount of assertions to ensure the correctness.
+We strongly recommend using a debug build of MMTk that includes all the debugging assertions
+when one is developing on a MMTk binding. The assertions are normal Rust `debug_assert!`,
+and they can be turned on in a release build with Rust flags (https://doc.rust-lang.org/cargo/reference/profiles.html#debug-assertions).
+
+## Extreme debugging assertions
+
+Other than the normal debugging assertions, MMTk also has a set of
```

Comment:
'Other than' is a bit ambiguous about whether 'normal debugging assertions' are included or not.

```suggestion
In addition to the normal debugging assertions, MMTk also has a set of
```

'Besides' should also be OK.